ape_safe/accounts.py

```diff
@@ -850,9 +862,11 @@ def skip_signer(signer: AccountAPI):
                 **gas_args,
                 nonce=submitter_account.nonce,
             )
-            txn = submitter_account.sign_transaction(exec_transaction, **signer_options)
+            if not (txn := submitter_account.sign_transaction(exec_transaction, **signer_options)):
```

Comment:
here is like the only real change- if the user skips signing, the txn is `None` here, so was getting an `AttributeError` on the next line when trying to set the sender.